docs/src/components/Utils/constants.ts

```diff
@@ -2,3 +2,10 @@ export const NEXT = 'next';
 export const PREVIOUS = 'previous';
 export const TITLE = 'title';
 export const COMPONENTS = 'COMPONENTS';
+export const COMPONENT_MIN_WIDTH = '47.5rem';
+
+export enum TEXT_ALIGNMENT {
+    center = 'center',
+    left = 'left',
+    right = 'right',
```

Comment:
`justify` is another option that exists, but I wonder if there's a global TS enum for this prop. lemme see what's out there.

Reply:
Doesn't look like it. We may want to consider those, but this is good enough to merge as is.